collect_app/src/main/java/org/odk/collect/android/storage/migration/StorageMigrationResult.java

```diff
@@ -6,19 +6,16 @@
 
 public enum StorageMigrationResult {
     SUCCESS,
-    FORM_UPLOADER_IS_RUNNING,
-    FORM_DOWNLOADER_IS_RUNNING,
     NOT_ENOUGH_SPACE,
-    MOVING_FILES_FAILED;
+    MOVING_FILES_FAILED,
+    CHANGES_IN_PROGRESS;
 
     public String getErrorResultMessage(Context context) {
         String errorMessage = context.getString(R.string.error) + " ";
         switch (this) {
             case NOT_ENOUGH_SPACE:
                 return errorMessage + context.getString(R.string.storage_migration_not_enough_space);
-            case FORM_UPLOADER_IS_RUNNING:
```

Comment:
I don't understand this.

Reply:
With the change lock we don't know which background service is running, but we know we shouldn't start the migration. I figured given these cases were rare it'd be ok to combine them into one error message?

Reply:
Whoops, this was a note to self I meant to come back to! But luckily you knew why I was confused.

Ok, I agree a single message is sufficient. I don't think that users know what "jobs" means in that context. How about "Automatic upload or download is running. Please try again later."